suite-native/app/src/StylesProvider.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { useColorScheme } from 'react-native';
+
+import { AppColorScheme, selectThemeVariant } from '@suite-native/module-settings';
+import { createRenderer, StylesProvider as TrezorStyleProvider } from '@trezor/styles';
```

Comment:
suite style provider?

Reply:
Probably something completely different is has nothing to do with Trezor or Suite, it's just styling provider... I will think about it

Reply:
I renamed it to StylesStyleProvider which makes makes more sense because it StyleProvider from "styles" lib.